Packages/ClientRuntime/Sources/Networking/Http/CRT/SDKDefaultIO.swift

```diff
@@ -14,32 +14,34 @@ import Darwin
 #endif
 
 public final class SDKDefaultIO {
```

Comment:
We should rename this type to something more descriptive and without "SDK" in the name. 
I'll file an issue to address this.